lib/components/artist/articles/index.js

```diff
@@ -29,7 +29,7 @@ class Articles extends React.Component {
 
 const styles = StyleSheet.create({
   heading: {
-    marginBottom: 20,
+    marginBottom: 12,
```

Comment:
Gotcha. Maybe it’s something we can revisit some time and figure out how to get those other components to not gobble up margin.